deployers/src/main/java/org/jboss/jca/deployers/fungal/RADeployer.java

```diff
@@ -215,6 +215,24 @@ public int getOrder()
          SecurityActions.setThreadContextClassLoader(oldTCCL);
       }
    }
+   
+   /**
+    * Get File object of URL who locate a rar resource
+    * @param url
+    * @return a File object pointing of rar resources
+    */
+   protected File getFileFromUrl(URL url) throws URISyntaxException, IOException{
+     return new File(url.toURI());
```

Comment:
Yes it is, in my case I search in Jar (like #535 )
I would add another one to clean tmp file like you said